content/strategy-goals/strategy/code-graph/code-intelligence/index.md

```diff
@@ -79,55 +79,54 @@ Recent key learnings:
 
-**Objective: Level up our enterprise-ready features**
+Taking it a step further, adding support for a wider set of artifactory hosts opens the door to a whole new array of cross-team features that support our use cases. These include (but are not limited to):
 
-- **KR:** Enable out-of-the-box cross-repo and dependency navigation for 3 customers by successfully moving them to on-prem auto-indexing.
-- **KR:** Improve precise code intelligence actions coverage by going from 3 to 5 languages (adding Kotlin and TS/JS), representing the languages in ~50% of all actions.
+- Accelerating [Developer onboarding](../use-cases/dev-onboarding.md) by enabling search for dependency versions and across transitive dependencies.
+- Monitoring [Code Health](../use-cases/code-health.md) by creating dependency tracking insights to detect deprecated dependency usage.
+- [Fixing security vulnerabilities](../use-cases/fixing-security-vulnerabilities.md) and helping [Incident response](../use-cases/incident-response.md) by searching and finding references to vulnerable dependency versions or errors, later applying a batch fix using Batch Changes.
+- Encourage [Code Reuse](../use-cases/code-reuse.md) by making precise navigation a medium for finding references to relevant usage examples.
 
-### Themes
+These are all currently in early stages of development and discussion. Read more about how we're thinking about the dependency graph in [RFC 593: Unlocking use cases based on the dependency graph of repositories](https://docs.google.com/document/d/1SkM8CG0IksvPEKRBRVLKipiRJTopx6Vq_hSWRJ9NyKs/edit#heading=h.trqab8y0kufp).
 
-**Cross repository and dependency navigation**
+**Ship precise language support**
 
-We believe this is the global code graph’s killer feature. It elevates the code navigation experience to a new level of cross-project analysis. It includes enabling precise cross-repository navigation and the ability to navigate to any third party dependency a repository references. We're solving this initially on Sourcegraph Cloud and plan to replicate the same functionality for on-premises usage.
+We’ve historically invested in broadening our span of supported languages. This is an ongoing effort that ties directly back to the Global Code Graph vision. We're currently focusing on shipping a revamped version of our JS/TS indexer and building a Python indexer.
 
 **Auto-indexing on-prem goes into Beta:**
 
-The current set up experience is not scalable for customers with a large amount of repositories. Enabling auto-indexing would mean a lower barrier for entry, a seamless experience and more engineers using precise code intelligence.
+The current set up experience is not scalable for customers with a large amount of repositories. Enabling auto-indexing would mean a lower barrier for entry, a seamless experience and more engineers using precise code intelligence. Setting up auto-indexing is also a requirement for Sourcegraph to be able to index dependency packages.
```

Comment:
Should we call out how strategically we're favouring accuracy over availability with precise (as in, to be precise we *have* to solve technical barriers that come with a cost) rather than availability over accuracy (going all in on a heuristic solution?).